oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/SampleFamily.java

```diff
@@ -233,12 +234,29 @@ public SampleFamily tag(Closure<?> cl) {
         }).toArray(Sample[]::new));
     }
 
+    /**
+     * Ignore histogram decrease bucket value, at agent side meter, need to ignore decrease histogram bucket value
+     */
+    public SampleFamily ignoreHistogramDecrease() {
+        this.context.ignoreHistogramDecrease = true;
+        return this;
+    }
+
+    /**
+     * Change default histogram bucket value unit,
+     * use second on the Prometheus, use millisecond on the meter side meter
+     */
+    public SampleFamily defaultHistogramBucketUnit(TimeUnit unit) {
```

Comment:
My preference is to add a new factory to build SampleFamily with those two settings. If we define functions in SampleFamily, that means end-users could access them, that will cause some potential issues. 

and they represent the different behaviors of the eco-system, I don't find any reason to access them in a MAL expression.

Reply:
Have created `SampleFactoryBuilder` to help create `SampleFactory` with these two settings.